Global Save Manager Wrist Monitor Data Storage

tfr8811 opened the pull request:
Global Save Manager now stores data for the wrist monitor
The save data is also now stored in Appdata

ajg8716 commented:
so the wrist monitor is there when you respawn. However, the information for the objectives is not updated on the wrist monitor anymore. Health and Stims work well though.

tfr8811 commented:
Fixed the serialization of Objectives. For some reason having {get; set;} after public variables made them unserializable. I also added [SerializeField] above the variables for clarity.

ajg8716 commented:
looks good and works great
